src/Routes/data.json

```diff
@@ -1,5 +1,6 @@
 {
 "type": "FeatureCollection",
+"date_range": {"start_date": "2022-05-20", "end_date": "2023-07-02"},
```

Comment:
Weird, but minor -- for consistency, "date_range" should be "dateRange" or you can make is simpler by just calling it "dates"

Reply:
Thanks, I've change this to `dates`, `start`, and `end`.